extensions/placeholder/package.json

```diff
@@ -13,16 +13,16 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "search-placeholder-images",
-      "title": "Search Placeholder Images",
+      "name": "search-placeholders",
```

Comment:
Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved